[DOCS] More doc updates to remove the CLI (#7874)
great-expectations/great_expectations: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.md

```

### 4. Confirm that the new Expectation Suites have been added

> You can confirm that Great Expectations can find your Expectation Suites by running the following Python code:

Reviewer: ```suggestion
To confirm that Great Expectations can find your Expectation Suites, run the following Python code:
```
